src/vm/constrainedexecutionregion.cpp

```diff
@@ -1262,6 +1262,7 @@ void PrepareCriticalFinalizerObject(MethodTable *pMT, Module *pModule)
     }
     _ASSERTE(pParent != NULL);
 
+#ifdef FEATURE_CER
```

Comment:
@jkotas Thanks for the suggestion. I've updated the PR to ifdef out `CerNgenRootTable::Restore` and `MethodCallGraphPreparer::Run`.